WindowsServerDocs/remote/remote-access/web-application-proxy/Publishing-Applications-with-SharePoint,-Exchange-and-RDG.md

```diff
@@ -75,7 +85,7 @@ If you want to restrict access to your Remote Access Gateway and add pre-authent
 
 #### How to publish an application in RDG using Web Application Proxy with pre-authentication
 
-1.  Web Application Proxy pre-authentication  with RDG works by passing the pre-authentication cookie obtained by Internet Explorer being passed into the Remote Desktop Connection client (mstsc.exe). This is then used by the Remote Desktop Connection client (mstsc.exe). This is then used by Remote Desktop Connection client as proof of authentication.
+1.  Web Application Proxy pre-authentication  with RDG works by passing the pre-authentication cookie obtained by Internet Explorer 11 or [Edge Chromium IE mode](/deployedge/edge-ie-mode) being passed into the Remote Desktop Connection client (mstsc.exe). This is then used by the Remote Desktop Connection client (mstsc.exe). This is then used by Remote Desktop Connection client as proof of authentication.
```

Comment:
Cosmetic/codestyle: I know, this is about something you did not change, but could we correct this spacing, since we are already editing this line?
```suggestion
1.  Web Application Proxy pre-authentication with RDG works by passing the pre-authentication cookie obtained by Internet Explorer 11 or [Edge Chromium IE mode](/deployedge/edge-ie-mode) being passed into the Remote Desktop Connection client (mstsc.exe). This is then used by the Remote Desktop Connection client (mstsc.exe). This is then used by Remote Desktop Connection client as proof of authentication.
```
(I might just pick up this file later for whitespace codestyle corrections, so feel free to skip this suggested change if you want to.